### DISCUSSION TITLE
UI translation

### DISCUSSION THREAD

**Discussion by Mejdunn** [pdfarranger/pdfarranger#1040] [Q&A] [answered]
Hy, I would like to send a Slovenian translation of UI, but I am not familiar with procedures on github.
Whom can I send file, to incorporate it in code and how can I check translations if I am Windows user.

Thank you very much
JG

**Comment by dreua** [comment]
I'm not sure I'll find the time to guide through the whole process but for starters you can just drop the file here in a comment if you already have it.

**Reply by dreua** [reply, marked as answer]
(I mean just attach it to your message or a comment here - drag and drop works or the second button below the comment field. It seems to work only for new comments.)

**Comment by Mejdunn** [comment]
[sl.zip](https://github.com/pdfarranger/pdfarranger/files/14030095/sl.zip)

Thank you very much. File in attachement

**Reply by dreua** [reply]
Build is running, once is done you can download the windows installer from the build artifacts and try it. Let me know if it works. #1041

**Reply by dreua** [reply]
here for this spedific build https://github.com/pdfarranger/pdfarranger/actions/runs/7668600940?pr=1041
You want the pdfarranger-windows-installer-msi

**Reply by Mejdunn** [reply]
Thank you, for your efford. It works great.
I had a quick skim over the interface and it seems to be mostly OK. I only found a few untranslated strings that I probably didn't have in the source .po file. I am attaching some screenshots below.
![Posnetek zaslona 2024-01-26 144022](https://github.com/pdfarranger/pdfarranger/assets/120342634/465a4b77-7396-42fa-8abc-f1102b6ed496)
![Posnetek zaslona 2024-01-26 144332](https://github.com/pdfarranger/pdfarranger/assets/120342634/7f1de38d-f7a3-4c07-bf21-f7375368dd97)
![Posnetek zaslona 2024-01-26 144252](https://github.com/pdfarranger/pdfarranger/assets/120342634/34939a03-9b7c-4d05-bf49-2162c27e91b4)
![Posnetek zaslona 2024-01-26 144136](https://github.com/pdfarranger/pdfarranger/assets/120342634/d5346016-cd21-4031-bcca-f228a24d1de8)

For the future, which .po file should I take next time if I want to update the translation?

Thank you in advance for your kindness.

**Reply by dreua** [reply]
You probably took the right one it just wasn't updated yet. But I am no expert on translations either.

**Reply by dreua** [reply]
I just updated the file with the scripts in the po directory and pushed it to the PR. Maybe you could try to edit that file further on Github, if you navigate to the changes from my pr there is a "three-dot-menu" on the top right of the file which should allow you to edit it. (Probably needs a new PR but that doesn't matter. Its still easier than sending the file around and me re-adding it in git.)

In general do not be afraid to just try stuff around here on Github, you don't have write access to our project so there is no risk in breaking anything serious :wink: 

Edit: Forgot the link to the changed file: https://github.com/pdfarranger/pdfarranger/pull/1041/files

**Reply by Mejdunn** [reply]
I have tried, but looks like I need permission for edtiting, since this option is grayed out.
![slika](https://github.com/pdfarranger/pdfarranger/assets/120342634/adb20076-53ab-432f-badd-44052a91db78)

Or are there some other dots, that I havent see.

**Reply by dreua** [reply]
Meh, sorry I didn't consider that this option doesn't work for you.

Once it is merged it should be easier to just edit the file. Feel free to send the file here again.

**Comment by Mejdunn** [comment]
I apologize for the inconvenience. I am attaching a file with completed translations.
[sl.txt](https://github.com/pdfarranger/pdfarranger/files/14067054/sl.txt)

**Reply by dreua** [reply]
I changed line 1 and 5, removed "plain" in the latter as I don't know what it means.

Do you want to test again before we merge it?

**Reply by Mejdunn** [reply]
Thank you very much for your diligence. 
I went over the changes and it seems OK. I don't see the need for further testing. 
Greetings
